app/src/main/res/drawable/ic_home.xml

```diff
@@ -4,6 +4,6 @@
     android:viewportHeight="24.0"
     android:viewportWidth="24.0">
     <path
-        android:fillColor="#FF000000"
+        android:fillColor="#555"
```

Comment:
Use tint color in image view instead of hardcoding

Reply:
Actually `ic_home` was not being used anywhere in the project and most of the drawables have the default colour `#555` that's why I made this change. Anyways I'll change it.

Reply:
If it is default, let it be `#555`. Just wanted to be consistent